other/materials_designer/specific_examples/optimization_film.ipynb

```diff
@@ -0,0 +1,412 @@
+{
```

Comment:
<p>There should be Ni and Gr in the title, file name and mkdocs link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>